dali/test/python/jax_plugin/utils.py

```diff
@@ -97,3 +99,30 @@ def sequential_pipeline_def():
         return data
 
     return sequential_pipeline_def()
+
+
+data_path = os.path.join(os.environ['DALI_EXTRA_PATH'], 'db', 'single', 'jpeg')
+
+
+def get_all_files_from_directory(dir_path, ext):
```

Comment:
out of curiosity. Any reason why you are not using the file root instead of listing every file?

Reply:
In the test for the JAX iterators I care a lot about the order of samples and things like that. So for example I test that with 2 GPUs dataset is sharded properly. 
To order the samples I use labels:
```python
file_names = get_all_files_from_directory(data_path, '.jpg')
file_labels = [*range(len(file_names))]
```
I was using an external source based pipeline before but I wanted to test it with "proper" reader. So I pass files and labels instead of `file_root` and then can check if the label (sample id) is as expected.